### ISSUE TITLE
URGENT Security Issue found 

### ISSUE THREAD

**Issue by chrisbarry** [JamesRamm/longclaw#472]
Hi all,

I've found a fairly serious security flaw with the platform I think? (it's not a python, django or wagtail issue, just data exposure) What's the best way we want to resovle this without alerting anyone publicly to the issue before it is fixed?

Does anyone have any guidance on this? How can we discuss this privately?

Chris

**Comment by JamesRamm** [comment]
I do not think this project is used in any serious production scenarios, so you can explain the issue here.
